encryption_formats.md

```diff
@@ -16,7 +16,7 @@ Note that encrypted data emitted is always bigger than the clear data provided.
 
 | **Element**        | **Buffer type** | **Byte length**   |
 | ------------------ | --------------- | ----------------- |
-| Version number (1) | Varint          | 1 byte            |
+| Version number (1) | Fixed length    | 1 byte            |
```

Comment:
Technically, I don't think we changed anything about v1. Probably best to leave it alone and forget it exists? =]

Reply:
True